### ISSUE TITLE
Add options for BigQuery dataset

### ISSUE THREAD

**Issue by ocervell** [terraform-google-modules/terraform-google-log-export#35]
Customers asked for 2 options:

- Set the table retention period for BigQuery dataset
- Create views while creating BQ dataset (to split prod / nonprod data for instance)

**Comment by kopachevsky** [comment]
@ocervell can you expand second item please:

Create views while creating BQ dataset (to split prod / nonprod data for instance)

How it should looks like from user standpoint tf configuration?

**Comment by kopachevsky** [comment]
@ocervell @aaron-lane regarding "Set the table retention period for BigQuery dataset", istn't it already in place?
https://github.com/terraform-google-modules/terraform-google-log-export/blob/32febc366365d285f183966a7c8706ba08211d3d/modules/bigquery/variables.tf#L44-L47

**Comment by ocervell** [comment]
@kopachevsky yep sorry, I don't know how I missed it. Creating views might be useful when creating org-level log exports, for instance to segregate environments in BigQuery (if environments are burried down the Business Unit > Team stack)

**Comment by kopachevsky** [comment]
@ocervell I just can't get idea how user will define view, because for current log export module when you define bigquery destination user don't even define any bigquery tables, only dataset, so if you have in mind

**Comment by aaron-lane** [comment]
@ocervell can you please respond?

**Comment by ocervell** [comment]
You can define view in the `google_bigquery_dataset` resource, as shown here: https://www.terraform.io/docs/providers/google/r/bigquery_dataset.html#view.

We would need to edit the `terraform-google-bigquery` module, so probably need to move this issue there.

**Comment by kopachevsky** [comment]
@ocervell cat you create issue in terraform-google-bigquery please?
